src/ex5.ml

```diff
@@ -0,0 +1,48 @@
+let loadXMLDoc () =
+  let xmlhttp = XmlHttpRequest.create () in
+  let onreadystatechange () =
+    match xmlhttp##readyState, xmlhttp##status with
+    | XmlHttpRequest.DONE, 200 ->
+      let xmlDoc = xmlhttp##responseXML in
+      let buf = Buffer.create 16 in
+      Buffer.add_string buf "<table border='1'><tr><th>Title</th><th>Artist</th></tr>";
```

Comment:
Avoid html-as-strings manipulation at all costs. We have `Tyxml_js` for that, use it.